src/test/java/seedu/address/logic/parser/AnakinEditCardParserTest.java

```diff
@@ -0,0 +1,162 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.INVALID_CARD_ANSWER_ARGS;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.INVALID_CARD_QUESTION_ARGS;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.VALID_ANSWER_A;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.VALID_ANSWER_B;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.VALID_CARD_ANSWER_ARGS;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.VALID_CARD_QUESTION_ARGS;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.VALID_QUESTION_A;
+import static seedu.address.logic.commands.AnakinCommandTestUtil.VALID_QUESTION_B;
+import static seedu.address.logic.parser.AnakinCommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.AnakinCommandParserTestUtil.assertParseSuccess;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ANSWER;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_QUESTION;
+import static seedu.address.testutil.AnakinTypicalIndexes.INDEX_FIRST_CARD;
+import static seedu.address.testutil.AnakinTypicalIndexes.INDEX_SECOND_CARD;
+import static seedu.address.testutil.AnakinTypicalIndexes.INDEX_THIRD_CARD;
```

Comment:
There is no `INDEX_THIRD_CARD`. Did you forget to commit that change?

Reply:
yes oops